Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "chel-fs"
+version = "0.1.0"
+edition = "2021"
```

Comment:
shouldnt this be 2024

Reply:
This may be some version, not year. It is generate by cargo.

Reply:
I will leave it as it is.